packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -137,22 +137,24 @@ function Cell(props) {
   }, []);
 
   /**
-   * Handles click event for cell
-   */
-  const onClick = useCallback(() => {
-    if (!isFocusTrapEnabled) {
-      onCellSelect({ rowId, columnId }, { row: rowIndex, col: columnIndex });
-    }
-  }, [isFocusTrapEnabled, onCellSelect, rowId, columnId, rowIndex, columnIndex]);
-
-  /**
-   * Hnadles the onDeactivate callback for FocusTrap component
+   * Handles the onDeactivate callback for FocusTrap component
    */
   const deactiveFocusTrap = () => {
     setFocusTrapEnabled(false);
     columnContext.setCellAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.resume-navigation' }));
   };
 
+  /**
+   * Handles mouse down event for cell
+   */
+  const onMouseDown = useCallback((event) => {
+    if (!isFocusTrapEnabled) {
+      onCellSelect({
+        rowId, columnId, rowIndex, columnIndex, multiSelect: !!event.shiftKey, selectedByKeyboard: false,
```

Comment:
Why are we using "!!" on event.shiftKey?  That is already a Booelan value.

Reply:
Updated